src/Sales/OrderBuilder.php

```diff
@@ -0,0 +1,143 @@
+use TddWizard\Fixtures\Customer\CustomerFixture;
+
+/**
+ * Builder to be used by fixtures
+ */
+class OrderBuilder
+{
+    /**
+     * @var CartBuilder
+     */
+    private $cartBuilder;
+
+    /**
+     * @var ProductBuilder[]
+     */
+    private $productBuilders;
+
+    /**
+     * @var CustomerBuilder
+     */
+    private $customerBuilder;
+
+    /**
+     * @var string
+     */
+    private $shippingMethod;
+
+    /**
+     * @var string
+     */
+    private $paymentMethod;
+
+    public static function anOrder(): OrderBuilder
+    {
+        return new static();
+    }
+
+    public function withProducts(ProductBuilder ...$productBuilders): OrderBuilder
+    {
+        $builder = clone $this;
+        $builder->productBuilders = $productBuilders;
+
+        return $builder;
+    }
+
+    public function withCustomer(CustomerBuilder $customerBuilder): OrderBuilder
+    {
+        $builder = clone $this;
+        $builder->customerBuilder = $customerBuilder;
+
+        return $builder;
+    }
+
+    public function withCart(CartBuilder $cartBuilder): OrderBuilder
+    {
+        $builder = clone $this;
+        $builder->cartBuilder = $cartBuilder;
+
+        return $builder;
+    }
+
+    public function withShippingMethod(string $shippingMethod): OrderBuilder
+    {
+        $builder = clone $this;
+        $builder->shippingMethod = $shippingMethod;
+
+        return $builder;
+    }
+
+    public function withPaymentMethod(string $paymentMethod): OrderBuilder
+    {
+        $builder = clone $this;
+        $builder->paymentMethod = $paymentMethod;
+
+        return $builder;
+    }
+
+    /**
+     * @return OrderInterface
+     * @throws \Exception
+     */
+    public function build(): OrderInterface
+    {
+        $builder = clone $this;
+
+        if (empty($builder->productBuilders)) {
+            // init simple products
+            for ($i = 1; $i <= mt_rand(1, 5); $i++) {
+                $builder->productBuilders[] = ProductBuilder::aSimpleProduct();
+            }
+        }
+
+        // create products
+        $products = array_map(
+            static function (ProductBuilder $productBuilder) {
+                return $productBuilder->build();
+            },
+            $builder->productBuilders
+        );
+
+        if (empty($builder->customerBuilder)) {
+            // init customer
+            $builder->customerBuilder = CustomerBuilder::aCustomer()
+                ->withAddresses(AddressBuilder::anAddress()->asDefaultBilling()->asDefaultShipping());
+        }
+
+        // log customer in
+        $customer = $builder->customerBuilder->build();
+        $customerFixture = new CustomerFixture($customer);
+        $customerFixture->login();
+
+        if (empty($builder->cartBuilder)) {
+            // init cart, add products
+            $builder->cartBuilder = CartBuilder::forCurrentSession();
+            foreach ($products as $product) {
+                $qty = mt_rand(1, 10);
```

Comment:
This could stay random, although a default of 1 seems more reasonable to me. Let's think about a way to parameterize the qtys and still keep the simple interface if qtys don't matter

Reply:
Maybe a `CartItemBuilder` could be used instead then, that allows for more flexibilty (also custom options etc)

And the order builder could still have `withProducts()` for the simple use case and  `withCartItems()` if more is needed.

But this is an additional feature request and won't keep me from merging the PR :)

Reply:
Oh I see the same is already possible with `withCart()` :+1: